coverage.sh

```diff
@@ -27,7 +27,7 @@ BINARIES=""
 for file in \
   $( \
     RUSTFLAGS="-C instrument-coverage" \
-      cargo +nightly test --tests --all --all-features --no-run --message-format=json \
+      cargo +nightly test --tests --all --all-features --no-run --message-format=json -- --include-ignored \
         | jq -r "select(.profile.test == true) | .filenames[]" \
         | grep -v dSYM - \
   ); \
```

Comment:
did you run this entirely? last 2 cleanup line gave me errors, thats why theyre different in wallet.rs
```
find . -name "*.profraw" -type f -delete &> /dev/null || true
find . -name "*.profdata" -type f -delete &> /dev/null || true
````

Reply:
It's been like that for a long time and working in the CI 🤷🏻

Reply:
probably due to the sub folder we have in iota.rs, coolio